core/merginsubscriptioninfo.cpp

```diff
@@ -111,6 +111,8 @@ void MerginSubscriptionInfo::setFromJson( QJsonObject docObj )
     emit planProductIdChanged();
   }
 
+  // check if some user action is required
+  mActionRequired = docObj.value( QStringLiteral( "action_required" ) ).toBool();
```

Comment:
As a rule of thumb, we only set property value when we know it has changed (and emit its specific signal):

```suggestion
  bool isActionRequired = docObj.value( QStringLiteral( "action_required" ) ).toBool();
  if ( isActionRequired != mActionRequired )
  {
    mActionRequired = isActionRequired;
  } 
```